ast/Trees.cc

```diff
@@ -304,7 +305,11 @@ optional<pair<core::SymbolRef, vector<core::NameRef>>> ConstantLit::fullUnresolv
     if (this->symbol != core::Symbols::StubModule()) {
         return nullopt;
     }
-    ENFORCE(this->resolutionScopes != nullptr && !this->resolutionScopes->empty(), "loc={}", this->loc.showRaw(ctx));
+    if (this->resolutionScopes == nullptr && this->resolutionScopes->empty()) [[unlikely]] {
```

Comment:
I think the old `||` was correct, you can't call `empty()` on `nullptr`.
```suggestion
    if (this->resolutionScopes == nullptr || this->resolutionScopes->empty()) [[unlikely]] {
```

Reply:
#8274